### PR TITLE
User can add plants to their plant list

### PR DESCRIPTION
**Pull request by marizaswart**
### Description
A user on the application is able to select the plants they would like to add to their personal plant list.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import ActionCable from 'actioncable'
 import './App.css';
 import {deAuthenticate} from "./Modules/Auth";
 
-const cable = ActionCable.createConsumer("ws://localhost:3001/cable");
+const cable = ActionCable.createConsumer("https://planted-api.herokuapp.com/cable");
```
Comment (tochman) [inline src/App.js]: ;-) What makes you think you can serve WebSockets through http?

File: src/App.js [hunk 2]
```diff
@@ -37,12 +37,12 @@ class App extends Component {
...
-                console.log('received message')
-                return toast(data.notification, {autoClose: 8000})
-            },
-        });
+        // cable.subscriptions.create({channel: 'NotificationChannel', user_id: JSON.parse(sessionStorage.getItem('current_user')).id}, {
+        //     received: (data) => {
+        //         console.log('received message')
+        //         return toast(data.notification, {autoClose: 8000})
+        //     },
+        // });
```
Comment (tochman) [inline src/App.js]: ??

Comment (marizaswart) [reply src/App.js]: Forgot to uncomment this :) can fix!

File: src/App.js (hunk 1 above)
Comment (marizaswart) [reply src/App.js]: :( can fix :)

**Pull request merged**